src/features/auth/services/metamask.service.ts

```diff
@@ -5,6 +5,7 @@ import BusinessError from '@metis/common/exceptions/business-error';
 import IMetaMaskService from './interfaces/metamask-service.interface';
 
 export default class MetaMaskService implements IMetaMaskService {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
It is worth to investigate if there is a type definition for ethereum object, so we don't use any type.